templates/confidential-computing/index.html

```diff
@@ -271,22 +274,39 @@ <h2>Enterprise-grade confidential VMs on all major public clouds</h2>
       </div>
     </section>
 
-    <hr class="p-rule--muted is-fixed-width" />
-
-    <section class="p-strip is-deep">
-      <div class="row">
-        <div class="col-12">
-          <p class="p-heading--2">
-            <a href="/engage/introduction-confidential-computing">Download our guide to confidential
-            computing&nbsp;&rsaquo;</a>
-          </p>
+    <section class="p-section">
+      <hr class="p-rule is-fixed-width" />
+      <div class="row--50-50">
+        <div class="col">
+          <h2>How to map security primitives<br> to attacker capabilities</h2>
+        </div>
+        <div class="col">
+          <div class="row p-image-container is-highlighted">
+          {{ image(url="https://assets.ubuntu.com/v1/d8d1c7c8-how-to-map.png",
+            alt="How to map security primitives to attacker capabilities",
```

Comment:
This is a decorative image so no need for alt text
```suggestion
            alt="",
```